.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -3,17 +3,17 @@
 ***Please submit all pull requests against the `development` branch. Failure to do so will delay or deny your request***
 
 - [] I have read and understood the [contributors guide](https://github.com/pi-hole/pi-hole/blob/master/CONTRIBUTING.md).
-- [] I have checked that [another pull request](https://github.com/pi-hole/pi-hole/pulls) for this purpose does not exist.
-- [] I have considered, and confirmed that this submission will be valuable to others.
-- [] I accept that this submission may not be used, and the pull request closed at the will of the maintainer.
-- [] I give this submission freely, and claim no ownership to its content.
+- [] I have written tests and verified that they fail without my change
```

Comment:
Will tests be required for every change? This may be optional, depending on the nature of the changes.